client/src/panels/panel_declarations/finances/welcome_mat/welcome_mat_exp_program_spending.js

```diff
@@ -134,7 +135,7 @@ export const format_and_get_exp_program_spending = (type, subject) => {
         return tick === _.first(plan_ticks) || tick === _.last(plan_ticks);
       } else {
         return (
-          tick === gap_year ||
+          tick === _.nth(all_ticks, _.round(all_ticks.length / 2)) ||
```

Comment:
This'll break if the number of history/planned years changes. Why the change?

Reply:
Because ticks look a bit bare empty with too few tick labels rendered. I'm simply rendering an additional tick label in the middle. How will this break?

Reply:
Ah, I thought this was specifically trying to dynamically locate the gap year for some reason. So it's just making sure that the label for the middle tick always renders? Ok.